staging/src/k8s.io/code-generator/_examples/HyphenGroup/informers/externalversions/factory.go

```diff
@@ -165,10 +165,19 @@ func (f *sharedInformerFactory) InformerFor(obj runtime.Object, newFunc internal
 	return informer
 }
 
+// StartWithStoptions implements the controller-manager InformerFactory interface.
+// Because the sharedInformerFactory is used only with builtin types, there is no reason why StartWithStopOptions
+// should ever be used.
+// Dynamicinformer and metadatainformer factories actually implement StartWithStopOptions.
+func (f *sharedInformerFactory) StartWithStopOptions(stopCh <-chan struct{}) {
+	f.Start(stopCh)
+}
+
 // SharedInformerFactory provides shared informers for resources in all known
 // API group versions.
 type SharedInformerFactory interface {
 	internalinterfaces.SharedInformerFactory
+	StartWithStopOptions(stopCh <-chan struct{})
```

Comment:
same question, could it be unified with Start(...)?

Reply:
done